src/aggregators/coevolve.jl

```diff
@@ -175,6 +175,7 @@ function next_time(p::CoevolveJumpAggregation{T}, u, params, t, i, tstop::T) whe
             rateinterval = get_rateinterval(p, lidx, u, params, t)
             if s > rateinterval
                 t = t + rateinterval
+                (t >= tstop) && break
```

Comment:
This wouldn’t work, because when you break you haven’t updated `_t` which is what is returned by the function.

Alternatively, this should give a correct trajectory:

```suggestion
                if t >= tstop
                    _t = t
                    break
                end
```